src/storage/src/hummock/compactor/mod.rs

```diff
@@ -629,14 +649,24 @@ impl Compactor {
                     };
 
                     match event {
-                        Some(Ok(SubscribeCompactionEventResponse { event })) => {
+                        Some(Ok(SubscribeCompactionEventResponse { event, create_at })) => {
                             let event = match event {
                                 Some(event) => event,
                                 None => continue 'consume_stream,
                             };
-
                             let shutdown = shutdown_map.clone();
                             let context = compactor_context.clone();
+                            let consumed_time_ms = SystemTime::now()
```

Comment:
Can we print a log if the consumed_time_ms is larger than expected (e.g. >500ms)?